crates/cli/src/dids/create.rs

```diff
@@ -0,0 +1,94 @@
+use clap::Subcommand;
+use std::sync::Arc;
+use web5::{
+    crypto::dsa::ed25519::{Ed25519Generator, Ed25519Signer},
+    dids::{
+        methods::{did_dht::DidDht, did_jwk::DidJwk},
+        portable_did::PortableDid,
+    },
+};
+
+#[derive(Subcommand, Debug)]
+pub enum Commands {
+    Jwk {
+        #[arg(long)]
+        no_indent: bool,
+        #[arg(long)]
+        json_escape: bool,
+    },
+    Web {
+        domain: String,
+    },
+    Dht {
+        #[arg(long)]
+        no_publish: bool,
+        #[arg(long)]
+        no_indent: bool,
+        #[arg(long)]
+        json_escape: bool,
+    },
+}
+
+fn print_portable_did(portable_did: PortableDid, no_indent: &bool, json_escape: &bool) {
+    let mut output_str = match no_indent {
+        true => serde_json::to_string(&portable_did).unwrap(),
+        false => serde_json::to_string_pretty(&portable_did).unwrap(),
+    };
+
+    if *json_escape {
+        output_str = output_str.replace('"', "\\\"");
+    }
+
+    println!("{}", output_str);
+}
+
+impl Commands {
+    pub fn command(&self) {
+        match self {
+            Commands::Jwk {
+                no_indent,
+                json_escape,
+            } => {
+                let private_jwk = Ed25519Generator::generate();
+                let mut public_jwk = private_jwk.clone();
+                public_jwk.d = None;
```

Comment:
yea these are strange, more data points that we need a public/private Jwk 'lite' type